README.md

```diff
@@ -2,7 +2,7 @@
 
 The following tutorial is intended to introduce you to many of the topics we'll be covering in Module 2. It's not expected that you fully understand each and every topic, and entirely expected that this might be your first introduction to all of them. Try to soak it in. We'll be talking more about these topics throughout the module.
```

Comment:
The instructions include 'you', 'we' and 'I'; I would try to re-format to use only one of those (probably 'you'). I'd be happy to help edit those, but not sure it's a huge deal.